fastlane/Fastfile

```diff
@@ -341,6 +341,18 @@ platform :mac do
     )
   end
 
+  desc 'Creates a new Kechain to use on UI tests'
+  lane :create_keychain_ui_tests do |options|
+    create_keychain(
+      name: "DefaultKeychain",
+      password: "default",
```

Comment:
Dominik, do you think it makes sense to create a random password even if we are not using it anywhere?

Reply:
Only to make it look more professional perhaps 😅  fine to keep it as is 👍